eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -0,0 +1,217 @@
+# Helper functions for retireving useful information from azure-sdk-for-* repo
+# Example Use : Import-Module .\eng\common\scripts\modules
+class PackageProps
+{
+    [string]$pkgName
+    [string]$pkgVersion
+    [string]$pkgDirectoryPath
+    [string]$pkgReadMePath
+    [string]$pkgChangeLogPath
+
+    PackageProps(
+        [string]$pkgName,
+        [string]$pkgVersion,
+        [string]$pkgDirectoryPath
+    )
+    {
+        $this.pkgName = $pkgName
+        $this.pkgVersion = $pkgVersion
+        $this.pkgDirectoryPath = $pkgDirectoryPath
+
+        if (Test-Path (Join-Path $pkgDirectoryPath "README.md")) {
+            $this.pkgReadMePath = Join-Path $pkgDirectoryPath "README.md"
+        } 
+        else {
+            $this.pkgReadMePath = $null
+        }
+
+        if (Test-Path (Join-Path $pkgDirectoryPath "CHANGELOG.md")) {
+            $this.pkgChangeLogPath = Join-Path $pkgDirectoryPath "CHANGELOG.md"
+        } 
+        else {
+            $this.pkgChangeLogPath = $null
+        }
+    }
+}
+
+Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
```

Comment:
Do you know if this is always available?

Reply:
It is being maintained / supported on the PowerShell gallery. The latest version was updated 14 days ago.